docs/config.toml

```
@@ -43,6 +43,11 @@ posts = "/:year/:month/:day/:title/"
   FlinkKubernetesOperatorStableShortVersion = "1.6"
   FlinkTableStoreStableVersion = "0.3.0"
   FlinkTableStoreStableShortVersion = "0.3"
+  ScalaVersion="2.12.7"
+  ScalaShortVersion="2.12"
+  JavaVersion="1.8"
+  Slf4jVersion="1.7.36"
+  Log4jVersion="2.17.1"
```

```suggestion
  ScalaVersion = "2.12.7"
  ScalaShortVersion = "2.12"
  JavaVersion = "1.8"
  Slf4jVersion = "1.7.36"
  Log4jVersion = "2.17.1"
```
Could we align the format with the other examples?